exporter/opentelemetry-exporter-otlp-proto-grpc/tests/test_otlp_trace_exporter.py

```diff
@@ -545,6 +550,55 @@ def test_span_status_translate(self):
             Status.DEPRECATED_STATUS_CODE_UNKNOWN_ERROR,
         )
 
+    # pylint:disable=no-member
```

Comment:
this tests makes me think we should make use to pytest parametrization https://docs.pytest.org/en/6.2.x/parametrize.html

Reply:
It could be a possible improvement however it might be a bit complex due to the nested property checking that this specific test is doing. I can see it being useful for other tests that have code branches like this one though.